lib/workers/repository/update/pr/index.ts

```diff
@@ -432,6 +452,7 @@ export async function ensurePr(
       }
       // TODO: types (#7154)
       logger.debug(`Created ${pr.displayNumber!}`);
```

Comment:
ah, we should change this log message to use `number` , like other logging's already do. can be done in separate PR

Reply:
I think we can remove the `displayNumber` property as it doesn't have any purpose left.